src/modules/Notifies/X11Notify.cpp

```diff
@@ -0,0 +1,147 @@
+/*
+	QMPlay2 is a video and audio player.
+	Copyright (C) 2010-2016  Błażej Szczygieł
+
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Lesser General Public License as published
+	by the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU Lesser General Public License for more details.
+
+	You should have received a copy of the GNU Lesser General Public License
+	along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <X11Notify.hpp>
+#include <QMPlay2Core.hpp>
+#include <Module.hpp>
+
+#include "dbus/notification.h"
+
+#include <QImage>
+
+#include <QDBusArgument>
+
+#define X11NotifyName "X11 Notify"
+
+QDBusArgument &operator<<(QDBusArgument &arg, const QImage &image)
```

Comment:
> I can compile & run without this, hmm... Is it necessary?

In previous versions of Qt it was needed to convert properly `QImage` -> dbus information. For now we don't send images at all in QMPlay2 (we send only text), but for future needs.